java/install.sh

```diff
@@ -11,3 +11,15 @@ update-alternatives --set javac /usr/lib/jvm/java-6-openjdk-amd64/bin/javac
 update-alternatives --set javaws /usr/lib/jvm/java-6-openjdk-amd64/jre/bin/javaws
 echo 'export JAVA_HOME=/usr/lib/jvm/java-6-openjdk-amd64' >> $HOME/.bashrc
 echo 'export PATH=$PATH:/usr/lib/jvm/java-6-openjdk-amd64/jre/bin' >> $HOME/.bashrc
```

Comment:
I think we should remove the two lines above this comment. We should have only one JAVA_HOME and one java in the path, and this should be Oracle 7 instead of openjdk 6